### PR TITLE
Import fix

### PR DESCRIPTION
**Pull request by deepak2016**
Fixes #

### Changes proposed in this pull request
Fixes import statements for the test java files. Removes imports with * and imports specific classes. Orders import statements correctly.
-

### Other links
-

### CODE REVIEW

**Comment by davidmoten** [comment]
This PR is screwed up and includes previously merged commits. Please fix. When you submit this PR you should only see mention of your commit.

File: src/main/java/com/microsoft/graph/requests/extensions/ChunkedUploadRequest.java [hunk 1]
```diff
@@ -111,7 +111,7 @@ public ChunkedUploadRequest(final String requestUrl,
                 result = this.baseRequest
                         .getClient()
                         .getHttpProvider()
-                        .send(baseRequest, ChunkedUploadResult.class, this.data, responseHandler);
+                        .send(baseRequest, (Class<ChunkedUploadResult<UploadType>>)(Class<?>) ChunkedUploadResult.class, this.data, responseHandler);
```
[the remark this answers is not in the document]
Comment (davidmoten) [reply src/main/java/com/microsoft/graph/requests/extensions/ChunkedUploadRequest.java]: was merged in Feb, this PR screwed up

**Comment by deepak2016** [comment]
Thanks David for taking time and reviewing the changes. I created "import-fix" branch from master and tried merging into "dev" branch. I will raise a separate pull request

File: src/main/java/com/microsoft/graph/requests/extensions/ChunkedUploadRequest.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (deepak2016) [reply src/main/java/com/microsoft/graph/requests/extensions/ChunkedUploadRequest.java]: I created "import-fix" branch from master and tried merging into "dev" branch. I will raise a separate pull request. Closing this pull request

[the remark this answers is not in the document]
Comment (deepak2016) [reply src/main/java/com/microsoft/graph/requests/extensions/ChunkedUploadRequest.java]: I thought we first checkin the changes into dev and then merge to masters, these changes seem to have been checked-into master directly.

**Comment by davidmoten** [comment]
@deepak2016  yep that would cause the problem!

**Pull request closed without merging**